### PR TITLE
Tuning attribute loader

### PR DESCRIPTION
**Pull request by beetlecrunch**

### CODE REVIEW

File: attributes-loader/src/main/resources/akka.conf [hunk 1]
```diff
@@ -1,5 +1,35 @@
-akka {
-  coordinated-shutdown {
-    exit-jvm = on
+kka {
```
Comment (TonioGela) [inline attributes-loader/src/main/resources/akka.conf]: TYPO

File: attributes-loader/src/main/resources/akka.conf [hunk 2]
```diff
@@ -1,5 +1,35 @@
...
+  coordinated-shutdown.exit-jvm = on
+  http {
+    server {
+      pipelining-limit = 128 # default 1
+      pipelining-limit = ${?PIPELINING_LIMIT}
+      backlog = 100 # default 100
+      backlog = ${?BACKLOG_SIZE}
+    }
```
Comment (galales) [inline attributes-loader/src/main/resources/akka.conf:9]: is this config required also if we do not use akka server?

**Review by galales — APPROVED** [review]

File: attributes-loader/src/main/resources/akka.conf (hunk 2 above)
Comment (TonioGela) [reply attributes-loader/src/main/resources/akka.conf:9]: good question: it should not.

**Comment by beetlecrunch** [comment]
The problem was related to the agreement process.
This PR is effectless.

**Pull request closed without merging**